hr_grade_rank/models/hr_grade_rank.py

```diff
@@ -0,0 +1,39 @@
+# See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class RankRank(models.Model):
+    _name = "rank.rank"
+    _description = "Rank"
+
+    name = fields.Char("Name")
+    description = fields.Text("Description")
+    salary_range = fields.Text("Salary Range")
```

Comment:
@ilakkia-serpentcs, you will have this warning (The attribute string is redundant. String parameter equal to name of variable) when you will run precommit.
so you can do like that:
name = fields.Char()
description = fields.Text(),
salary_range = fields.Text()
so no need to put again string value in this case.

Reply:
yes  removed string as it is deprecated